chore: update PolyScale integration info (#12810)
supabase/supabase: apps/docs/pages/guides/integrations/polyscale.mdx

  ></iframe>
</div>

<Admonition type="note">

> PolyScale presently provides caching for TCP connections to Supabase. If you wish to make an http connection to Supabase, you can use GraphQL, for which PolyScale also provides support. Support for caching with the Supabase client is coming soon. For more information, please [contact PolyScale](https://www.polyscale.ai/contact/).

Reviewer: ```suggestion
PolyScale provides caching for TCP connections and GraphQL. Support for caching with the Supabase client is coming soon. 
```
